src/styles/_catalog.scss

```diff
@@ -1,9 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
```

Comment:
This width looks like main value that can be reused, let's create variable for it